sources/background_subtractor/bgsubstractor.cpp

```diff
@@ -16,19 +17,17 @@ namespace tmd {
         tmd::debug("BGSubstractor", "BGSubstractor", "bgs created.");
 
         m_input_video_path = video_folder + "ace_" + std::to_string
-                                                 (camera_index) + ".mp4";
+                (camera_index) + ".mp4";
 
         std::string mask_path = tmd::Config::mask_folder + "mask_ace" +
-                std::to_string(camera_index) + ".jpg";
+                                std::to_string(camera_index) + ".jpg";
         m_static_mask = cv::imread(mask_path, 0);
 
-        m_input_video.open(m_input_video_path);
-
-        cv::Mat bg;
-        m_input_video.read(bg);
+        std::string background_path = tmd::Config::bgs_empty_room_background
+                          + "/ace_" + std::to_string(camera_index) + ".jpg";
+        cv::Mat bg = cv::imread(background_path);
```

Comment:
Even if there is a big blob, it is not taken into account when separating the blobs, so there is no speed up nor down.